ext/pdo_dblib/tests/types.phpt

```diff
@@ -64,3 +69,5 @@ bool(true)
 bool(true)
 bool(true)
 bool(true)
+bool(true)
+bool(true)
```

Comment:
This fails for me. The column type comes back as SQLBINARY (45), so it's handled differently.